components/automate-ui/src/app/entities/service-groups/service-groups.reducer.ts

```diff
@@ -59,7 +60,8 @@ export const ServiceGroupEntityInitialState: ServiceGroupsEntityState = {
   suggestions: {
     values: [],
     status: EntityStatus.notLoaded
-  }
+  },
+  serviceDeletionStatus: EntityStatus.notLoaded
```

Comment:
I added a status specifically for the deletion because there are other elements to the page that are loading and this allows to me separate out a success/failure notification for deletion more clearly.